### PR TITLE
Update DNS entries to use better ones

### PR DESCRIPTION
**Pull request by rpsene**
This PR changes the DNS servers used, adding a fastest one and the one provided by IBM.

### CODE REVIEW

File: var.tfvars [hunk 1]
```diff
@@ -30,7 +30,7 @@ pull_secret_file            = "data/pull-secret.txt"
 cluster_domain              = "example.com"
 cluster_id_prefix           = "test"
 
-dns_forwarders              = "8.8.8.8; 8.8.4.4"
+dns_forwarders              = "1.1.1.1; 9.9.9.9"
```
Comment (bpradipt) [inline var.tfvars:33]: Is the 9.9.9.9 public ?

Comment (rpsene) [reply var.tfvars:33]: @bpradipt 
yep, it is https://www.quad9.net/

Comment (bpradipt) [reply var.tfvars:33]: Thanks @rpsene

**Review by bpradipt — APPROVED** [review]
lgtm

**Pull request merged**